packages/conventional-changelog-core/lib/merge-config.js

```diff
@@ -264,9 +267,27 @@ function mergeConfig(options, context, gitRawCommitsOpts, parserOpts, writerOpts
                 } else {
                   context.currentTag = context.currentTag || firstCommitHash;
                 }
-              } else {
-                if (!context.currentTag) {
-                  context.currentTag = options.lernaPackage ? options.lernaPackage + '@' + context.version : 'v' + context.version;
+              } else if (!context.currentTag) {
```

Comment:
not a blocker; but there's gettin to be enough logic here, it might be worth pulling this into a helper ... guessNextTag()?